scripts/suse/zypper/plugins/commit/zyppnotify

```diff
@@ -19,7 +19,9 @@ class DriftDetector(Plugin):
     def __init__(self):
         Plugin.__init__(self)
         self.ck_path = "/var/cache/salt/minion/rpmdb.cookie"
-        self.rpm_path = "/var/lib/rpm/Packages"
+        self.rpm_path = "/var/lib/rpm/Packages.db"
```

Comment:
later some methods read from this file in specific places, is this ndb compatible?

Reply:
Packages.db in this case is ndb file, it should be preferred one if exists. I just discussed it with @mlschroe.

Reply:
Indeed, but as https://github.com/openSUSE/salt/pull/304/files/42724735d506be32882f3d91a22f456354ef3a8a#diff-de57eeeae39d1af47a8c4bbbac040ff8fc6a38bab688dc1f72f755a148902cf8R43 we read a section from there, but I guess that is not binary compatible both formats?

Reply:
I see what you mean, but zyppnotify is reading not just one section of 0x1000 bytes from the file, it is reading it whole by blocks with size equal to 4096 and the cookie file contains sha256 hash of whole file.

Reply:
Ouch. Indeed, yes!